### DISCUSSION TITLE
Allow an Organization member to see the Team list

### DISCUSSION THREAD

**Discussion by Franra** [grafana/grafana#41007] [General]
Hello,

Version: 
Grafana v8.1.7 (34b8880d53)
[database] sqlite3 and postgres (not tested with mysql)

I'm not really sure if this is a bug or a feature.
But when a user is Viewer in an Organization and Admin on a Folder.
In Folder Permissions, he can't  Add Permission For a Team (because he only see is own team on the list).
However he can Add Permission For a User on another Team. 

**It would be nice if a User can add a Team instead of adding Users one by one.**

To illustrate my point :
Imagine there is two Teams.
![image](https://user-images.githubusercontent.com/33827929/139107799-89f249ca-6ac5-4121-9b7d-1f76f3920edf.png)

And each one as a Folder (with Admin right on it).
![image](https://user-images.githubusercontent.com/33827929/139109872-8ad8dfbd-87dc-4d65-ad28-93caeb2952c6.png)

The members of the Team "Number" can't see the existence of the Team "Letter" to add rights (and vice versa)
![image](https://user-images.githubusercontent.com/33827929/139109327-6215727b-26df-439e-8983-6723675ed8a3.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
